components/dashboard/src/start/CreateWorkspace.tsx

```diff
@@ -152,6 +156,19 @@ export default class CreateWorkspace extends React.Component<CreateWorkspaceProp
   }
 }
 
+function LimitReachedModal(p: { children: React.ReactNode }) {
+  return <Modal visible={true} closeable={false} onClose={()=>{}}>
+    <h3>Limit Reached</h3>
```

Comment:
**question**: How easy could be to incorporate the account avatar here so that it's easier to recognize the account associated, as seen in [the specs](https://github.com/gitpod-io/gitpod/issues/3301)? It could make this message also feel a bit personal. What do you think? Your call.

Reply:
Aha, thanks for pointing that out! Will try.